src/components/atoms/UserStatusDropdown/UserStatusDropdown.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { UserStatus } from "types/User";
+import { useShowHide } from "hooks/useShowHide";
+
+import "./UserStatusDropdown.scss";
+
+export interface UserStatusDropdownProps {
+  options: Array<UserStatus>;
+  onChange: (option: UserStatus) => void;
+  label: UserStatus | string;
```

Comment:
If it's just the label of Dropdown, should it be just string? Or am I missing something?

Reply:
There is only limited values could be here